mcf-browser/src/back-end/utils.js

```diff
@@ -42,10 +42,19 @@ const ERROR_MESSAGES =
  * @return {Object} An object containing both 'in' and 'out' property labels.
  */
 async function getRemotePropertyLabels(dcid) {
-  const targetUrl = API_ROOT + '/node/property-labels?dcids=' + dcid;
-  return fetch(targetUrl)
+  // Get outward property labels
+  const outTargetUrl = API_ROOT + '/v1/properties/out/' + dcid;
+  const outPropertyLabels = await fetch(outTargetUrl)
+      .then((res) => res.json())
+      .then((data) => data.properties);
+
+  // Get inward property labels
+  const inTargetUrl = API_ROOT + '/v1/properties/in/' + dcid;
+  const inPropertyLabels = await fetch(inTargetUrl)
       .then((res) => res.json())
-      .then((data) => JSON.parse(data.payload)[dcid]);
+      .then((data) => data.properties);
```

Comment:
You can fetch in and out properties concurrently like https://stackoverflow.com/a/59664248

Reply:
Changed in next commit!